portal/views/assessment_engine.py

```diff
@@ -1801,6 +1807,10 @@ def patient_assessment_status(patient_id):
     if trace:
         establish_trace(
             "BEGIN trace for assessment-status on {}".format(patient_id))
+
+    purge = request.args.get('purge', False)
+    if purge == 'True':
```

Comment:
would prefer a case-insensitive check
```suggestion
    if purge.lower() == 'true':
```